admin/class-purger.php

```diff
@@ -403,8 +403,9 @@ protected function delete_cache_file_for( $url ) {
 		 * @since 2.1.0
```

Comment:
Add `@since 2.2.3` which could be upcoming version and mention which additional param passed to this filter.

Reply:
Can you please check whether it is good? [https://prnt.sc/6wsppOSb2UIC](https://prnt.sc/6wsppOSb2UIC)

Reply:
Add below line after `@since 2.1.0`.

```
@since 2.2.3 Purge URL argument `$url` were added.
```